steps/jira-issue-update-build/0.1.3/step.yml

```diff
@@ -0,0 +1,82 @@
+title: Update JIRA issues with build number
+summary: Update fields of JIRA issues with build number.
+description: "Update fields of JIRA issues associated with current MR/PR, with the
```

Comment:
This seems very similar to the step [jira-issue-field-add-or-update](https://app.bitrise.io/integrations/steps/jira-issue-field-add-or-update). What is the different functionality that warrants a _separate_ step? Have you considered extending that step?